src/common/transformations/src/transformations/op_conversions/convert_ti_to_sequences.cpp

```diff
@@ -412,8 +418,856 @@ ov::pass::ConvertTensorIteratorToGRUSequence::ConvertTensorIteratorToGRUSequence
     register_matcher(m, callback);
 }
 
+static bool get_scalar_constant_value(const ov::Output<ov::Node>& node, int64_t& output_value) {
```

Comment:
do we have a helper function for this in utils files?

Reply:
There are not this function in utils. I would like to extend utils by this function, because I need to do the same action in my pr. @itikhono What do you think about the suggestion?